docs/services/rumqttd.md

```diff
@@ -0,0 +1,37 @@
+# rumqttd
+
+[rumqttd](https://github.com/bytebeamio/rumqtt/tree/main/rumqttd) is a high performance, embeddable [MQTT](https://en.wikipedia.org/wiki/MQTT) broker installed via [mother-of-all-self-hosting/ansible-role-rumqttd](https://github.com/mother-of-all-self-hosting/ansible-role-rumqttd).
```

Comment:
Isn't it better to link to https://github.com/bytebeamio/rumqtt instead of to https://github.com/bytebeamio/rumqtt/tree/main/rumqttd?

This applies to `supported-services.md` as well.